tests/testthat/test-parsing.R

```diff
@@ -36,10 +36,17 @@ test_that("mixed CRLF / LF EOLs fail", {
   )
 })
 
-test_that("unicode can't be propprely handled on Windows for R < 4.2", {
-  msg <- ifelse(getRversion() < "4.2" && is_windows(),
-    "Can't parse input due to unicode restriction in base R\\.",
-    NA
+test_that("unicode can't be properly handled on Windows for R < 4.2", {
```

Comment:
I think splitting this test into two makes both of them a bit more readable, but I don't feel too strongly about this change. WDYT?

Reply:
Good idea yes.